tests/test_kuzmin.py

```diff
@@ -137,22 +138,52 @@ def test_K1_function_of_temperature():
 
 
 def test_kuzmin_properties():
-    """Test the kuzmin_properties function."""
+    """Test the kuzmin_properties function.
+
+    We check that the all the attributes are defined and are of the right type.
+    We check that if K1_0 is not given, then the attribute K1 is None.
+    We check that an error is raised if the Ms_0 is not given either as an input
+    or in the data vector Ms.
+    We check that the function works as intended both giving Tc and not.
```

Comment:
I think we should split this into four separate functions, each testing one of the cases listed here.